### PR TITLE
adding-oic-token-exchange-demo

### PR DESCRIPTION
**Pull request by pdtoal**
Creating a new tutorial and sample code to show how to exchange identity tokens to trusted tokens that can be used to call IAM-protected services. Created a new shared-assets folder under CISO folder as requested.

### CODE REVIEW

File: security/ciso-office/shared-assets/oic_iam_token_exchange_demo/environment_template.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#VARIABLES TO CHANGE FOR YOUR ENVIRONMENT
+#########################################
+
+#IDCS Instance Variables
+idcs_url = "" # e.g. "https://idcs-1234.identity.oraclecloud.com/" 
+idcs_client_id="" # e.g. e123432ejcn43k2kdkrkfkke
+idcs_client_secret=""# e.g. e123432-ejcn-43k2-kdkr-kfdswe33kke
+
+#Endpoint audience that you will be requesting an IDCS token for, e.g. "https://EAFE304003E34FA56.integration.ocp.oraclecloud.com:443"
```
Comment (oheimburger) [inline security/ciso-office/shared-assets/oic_iam_token_exchange_demo/environment_template.py:9]: Is this a real audience URL?

File: security/ciso-office/shared-assets/oic_iam_token_exchange_demo/environment_template.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+idcs_scope=target_audience + ""
+
+#Target Instance Variables
+#FQDN of the target host you will be calling with your IDCS token, e.g. "https://oic-fr2323hpc-fr.integration.ocp.oraclecloud.com"
+target_host="" 
+
+# URI of your target endpoint, e.g. "/ic/api/integration/v1/flows/rest/ECHO/1.0/Testing"
+target_url= target_host + ""
+
+#Scope of your registered application which you will request in your IDCS token, e.g. "https://EAFE304003E34FA56.integration.ocp.oraclecloud.com:443urn:opc:resource:consumer::all"
```
Comment (oheimburger) [inline security/ciso-office/shared-assets/oic_iam_token_exchange_demo/environment_template.py:22]: See my comment for line 9.

**Review by oheimburger — CHANGES_REQUESTED** [review]
Update the screen shots for the new OCI AIM UI.

File: security/ciso-office/shared-assets/oic_iam_token_exchange_demo/environment_template.py (hunk 2 above)
Comment (pdtoal) [reply security/ciso-office/shared-assets/oic_iam_token_exchange_demo/environment_template.py:22]: No, it's not real

**Review by oheimburger — APPROVED** [review]

**Pull request merged**